donkeycarManagerFront/src/views/registerView.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="main">
+    <va-alert v-if="success === true" color="success" class="mb-4 alert" :title="'Enregistrement réussis'"> Vous pouvez vous diriger vers l'écran pour piloter une voiture !</va-alert>
+    <va-alert v-if="success === false" color="danger" class="mb-4 alert" :title="'Impossible de vous enregistrer !'"> Veuillez réessayer ou bien demander de l'aide à un responsable du stand</va-alert>
+    <h1 class="mainTitle">Inscris-toi pour piloter 🏎️ </h1>
+    <RegisterForm :runTime="listOption" @success="success=$event" class="form"></RegisterForm>
+  </div>
+  <div class="option-container">
+      <runOption :listOptions="listOption" @newParam="pushParam($event)"></runOption>
+    </div>
+</template>
+
+<script>
+
+import RegisterForm from '@/components/form.vue'
+import runOption from '@/components/runOption.vue'
+
+export default {
+  components: {
+    RegisterForm,
+    runOption
+  },
+  data () {
+    return {
+      success: null,
+      listOption: {}
+    }
+  },
+  watch: {
+    success: function () {
+      const that = this
+      setTimeout(function () { that.success = null }, 4000)
+    }
+  },
```

Comment:
change la fonction en fonction fléchée et pas besoin de faire un that = this juste au dessus

Reply:
t'es sur, le linter a pas l'air de kiffer : 
`/home/yannis/Documents/pro/donkeycarLPH/donkeycarManagerFront/src/views/registerView.vue
  30:14  error  You should not use an arrow function to define a watcher  vue/no-arrow-functions-in-watch
`

Reply:
c'est pou ça que j'avais pas utilisé une fonction fléchée

Reply:
```js
watch: {
  success () {
    setTimeout(() => this.success = null, 4000)
  }
}
```

Reply:
Ah, merci chef 😂

Reply:
```
  31:18  error  Arrow function should not return assignment  no-return-assign

```
en cours de fix